integration/token/common/sdk/identity/provider.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package identity
+
+import (
+	"github.com/hyperledger-labs/fabric-token-sdk/token"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/db/driver"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/identity"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/identity/storage/kvs"
+)
+
+type KVSStorageProvider struct {
```

Comment:
To put things into perspective, we could implement this interface that exists already and is implemented by `unity`, `sql` and `memory`. Then KVS would be an alternative implementation to it (of course what is not implemented can panic, but it will still be clear what it is about).
```
type Driver interface {
	NewTokenLock(opts common.Opts) (TokenLockDB, error)

	NewWallet(opts common.Opts) (WalletDB, error)

	NewIdentity(opts common.Opts) (IdentityDB, error)

	NewToken(opts common.Opts) (TokenDB, error)

	NewTokenNotifier(opts common.Opts) (TokenNotifier, error)

	NewAuditTransaction(opts common.Opts) (AuditTransactionDB, error)

	NewOwnerTransaction(opts common.Opts) (TokenTransactionDB, error)
}
```

Reply:
Otherwise maybe we don't need the walletdb and the identitydb, since we could use the memory implementation instead.

Reply:
good point, please, open a github issue for this 👍